doc/dev.md

```diff
@@ -38,16 +38,19 @@ Flags can be struct with bit-fields.
   - Use commit messages with a context prefix to make it easier to find and understand, ex:<br>
   `mp3: Validate sync correctly`
 - Tests:
-  - If possible use a pair of `testdata/file` and `testdata/file.fqtest` where `file.fqtest` is `$ fq dv file` or `$ fq 'dv,torepr' file` if there is `torepr` support.
+  - If possible, add one or more pairs of example input file and expected CLI output, with naming like:
+    - `./format/<format_name>/testdata/<name>.<ext>`, e.g. [`./format/mp4/testdata/aac.mp4`](../format/mp4/testdata/aac.mp4)
+    - and `./format/<format_name>/testdata/<name>.fqtest`, e.g. [`./format/mp4/testdata/aac.fqtest`](../format/mp4/testdata/aac.fqtest)
+  - The latter contents should be `$ go run . dv <file_path>` or `$ go run . 'dv,torepr' <file_path>` if there is `torepr` support.
```

Comment:
Should probably be `$ fq ...`. The `.fqtest` files looks very much like shell scripts but they are more like "transcripts" that the fq test system deserialize/serialize and execute using the go test system, so no there is no external processes executed per test etc.